packaging/Description.txt

```diff
@@ -0,0 +1 @@
+placeholder
```

Comment:
This file and welcome.txt seem like oversights

Reply:
Being that these files are intended for the GUI installers, is there an existing resource that would serve as the description and welcome texts? If not, it looks like it's just going to generate noise so I'll keep it out of the PR.

Reply:
I've decided to delete these files and run with the default generated texts.